python/nav/portadmin/snmp/base.py

```diff
@@ -45,6 +46,28 @@
 _logger = logging.getLogger(__name__)
 
 
+def translate_protocol_errors(func):
+    """Decorator that translates SNMPErrors into PortAdmin ProtocolErrors.
+
+    The PortAdmin API will handle ProtocolErrors gracefully, but other exceptions
+    will bleed through and be handled by Django's 500 handler.
+    """
+
+    @wraps(func)
+    def _wrapper(*args, **kwargs):
+        try:
+            return func(*args, **kwargs)
+        except SnmpError as error:
```

Comment:
"nmp" is actually lowercase in the exception name? <insert quip about library quality here>

Reply:
Yes *grumble*